pkgs/by-name/dn/dnsdbq/package.nix

```
@@ -0,0 +1,56 @@
+{
+  curl,
+  dnsdbq,
+  fetchFromGitHub,
+  jansson,
+  lib,
+  nix-update-script,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "dnsdbq";
+  version = "2.6.7";
+
+  src = fetchFromGitHub {
+    owner = "dnsdb";
+    repo = "dnsdbq";
+    rev = "v${version}";
+    hash = "sha256-VeoLgDLly5bDIzvcf6Xb+tqCaQxIzeSpoW3ij+Hq4O8=";
+  };
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = {
+      version = testers.testVersion {
+        package = dnsdbq;
+        command = "dnsdbq -v";
+      };
+    };
+  };
+
```

```suggestion

  nativeBuildInputs = [
    curl # curl-config
  ];

```
This will be required once #178468 is implemented, so better have it fixed now.

---

Thanks, I've added it.